pkgs/tools/virtualization/awsebcli/default.nix

```diff
@@ -0,0 +1,99 @@
+ { stdenv, python }:
+let
+  inherit (localPythonPackageSet) pkgs;
+  inherit (pkgs) fetchPypi buildPythonApplication;
+
+  localPythonPackageSet = python.override {
+    packageOverrides = self: super: rec {
+      colorama = super.colorama.overridePythonAttrs (oldAttrs: rec {
+        version = "0.3.7";
+
+        src = fetchPypi {
```

Comment:
Note that `fetchPypi` is overridable, so you can write 
```
src = oldAttrs.src.override { inherit version; sha256 = "..."};
```
No need to change it, though.